src/main/java/lotto/Main.java

```diff
@@ -1,20 +1,23 @@
 package lotto;
 
+import lotto.domain.Amount;
 import lotto.domain.Lotto;
 import lotto.domain.Lottos;
+import lotto.domain.Number;
 import lotto.view.InputView;
 import lotto.view.ResultView;
 
 public class Main {
     public static void main(String[] args) {
         Lottos lottos = purchageLottos();
         Lotto winningNumbers = inputWinningNumbers();
-        printReport(lottos, winningNumbers);
+        Number bonusNumber = inputBonusNumber();
+        printReport(lottos, winningNumbers, bonusNumber);
```

Comment:
```suggestion
        Ranks ranks = lottos.getRanks(winningNumbers, bonusNumber);
        ResultView.printReport(ranks);
```

함께다녀야하는 변수가 많아지면서, 복잡해지는것 같기도해요 🤔 
새로운 객체가 없을까요?

Reply:
> 다만 마지막 커밋에 로또 2등 구현이 거의 들어가 있는데요. 원래는 요구사항 나눈 것마다 커밋을 하려고 했으나, 2등 추첨하는 코드를 작성하니 연쇄적으로 입력과 출력 관련 코드에서 에러가 나서 다같이 수정하게 되었습니다; 테스트 하기 어려운 코드가 돼서 그런 건가요?

[인자개수](https://www.google.com/search?q=%ED%81%B4%EB%A6%B0%EC%BD%94%EB%93%9C+%EC%9D%B8%EC%9E%90%EA%B0%9C%EC%88%98) 한번 다양한 포스팅을 읽어보시면 좋을것 같습니다~

Reply:
우승번호와 보너스번호를 갖고 있는 WinningLotto 클래스를 추가해봤습니다!
감사합니다.